### PR TITLE
Skills section

### PR DESCRIPTION
**Pull request by StefanBehring**
Add contact section
Edit css header
Edit css nav-links
Edit css colors at different places

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -69,10 +72,34 @@ <h3 class="projects-project--title">Gfx Checker</h3>
...
+                    <img class="projects-skills--image" src="img/react.svg" alt="react">                    
+                </div>
             </section>
         </article>
-
+        <article id="contact" class="contact">
+            <h2 class="contact-title">Contact</h2>
+            <form class="contact-form" action="#" method="POST">
+                <label class="contact-form--label" for="name">Name</label>
+                <input class="contact-form--element" type="text" name="name">
```
Comment (macfrei) [inline index.html:95]: Please check out how to properly connect inputs and labels (hint: you are missing an `id` in the input).

File: index.html [hunk 2]
```diff
@@ -69,10 +72,34 @@ <h3 class="projects-project--title">Gfx Checker</h3>
...
+                    <img class="projects-skills--image" src="img/html5.svg" alt="html5">
+                    <img class="projects-skills--image" src="img/java.svg" alt="java">
+                    <img class="projects-skills--image" src="img/javascript.svg" alt="javascript">
+                    <img class="projects-skills--image" src="img/mongodb.svg" alt="mongodb">
+                    <img class="projects-skills--image" src="img/mysql.svg" alt="mysql">
+                    <img class="projects-skills--image" src="img/nodejs.svg" alt="nodejs">
+                    <img class="projects-skills--image" src="img/php.png" alt="php" width="512" height="512"> 
+                    <img class="projects-skills--image" src="img/python.svg" alt="python">
+                    <img class="projects-skills--image" src="img/react.svg" alt="react">                    
+                </div>
```
Comment (macfrei) [inline index.html:88]: I think it would be better semantically if this is a list 🙂

File: css/styles.css [hunk 3]
```diff
@@ -178,4 +211,68 @@ h1 {
...
+    width: 80%;
+    max-width: 400px;
+}
+
+.contact-form--label {
+    color: var(--thi-color-4);
+}
+
+.contact-form--element {
+    background-color: var(--main-color-1);
```
Comment (macfrei) [inline css/styles.css:258]: It's cool that you are using CSS variables. Maybe think about a better naming convention? Right now I wouldn't know what kind of color this is..

File: css/styles.css [hunk 4]
```diff
@@ -178,4 +211,68 @@ h1 {
...
+}
+
+.projects-skills--title {
+    color: var(--main-color-1);
+    margin-left: 2rem;
+}
+
+.projects-skills--overview {
+    align-items: center;
+    display: flex;
```
Comment (macfrei) [inline css/styles.css:223]:
```suggestion
    display: flex;
    align-items: center;
```

**Review by macfrei — COMMENTED** [review]
Well done! You have a lot of advanced features going on here 🆒 👍

File: index.html (hunk 1 above)
Comment (StefanBehring) [reply index.html:95]: Ok, I think I have them correct now.

File: css/styles.css (hunk 3 above)
Comment (StefanBehring) [reply css/styles.css:258]: This is something I am interested in, like in general how to properly name classes aso in css

Comment (macfrei) [reply css/styles.css:258]: We will also discuss it next week in more detail :)

**Pull request merged**